dask/dataframe/_compat.py

```diff
@@ -221,3 +223,7 @@ def is_any_real_numeric_dtype(arr_or_dtype) -> bool:
             and not is_complex_dtype(arr_or_dtype)
             and not is_bool_dtype(arr_or_dtype)
         )
+
+
+def get_numeric_only_kwargs(numeric_only) -> dict:
+    return {} if numeric_only is no_default else {"numeric_only": numeric_only}
```

Comment:
Wasn't sure, happy to move over though.

Reply:
Sounds good -- I went ahead and pushed a small commit that moves `no_default` back and `get_numeric_only_kwargs` to `utils.py`. Figured it'd be good to get this in sooner so it can be used in other `numeric_only` PRs

Reply:
Sounds good